build.gradle

```diff
@@ -336,7 +386,8 @@ project(':iceberg-hive') {
     compile project(path: ':iceberg-core', configuration: 'shadow')
     compile project (path: ':iceberg-api', configuration: 'shadow')
     compile project (path: ':iceberg-common', configuration: 'shadow')
-    compile project (path: ':iceberg-mr', configuration: 'shadow') 
+    compile project (path: ':iceberg-mr', configuration: 'shadow')
+    compile project (path: ':iceberg-parquet', configuration: 'shadow')
```

Comment:
Is this really needed? I'm just wary of adding more cross-subproject dependencies.

Reply:
Same

Reply:
^ both these needed